src/components/MenuBar/ButtonBar.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import Button from '@material-ui/core/Button';
+import PropTypes from 'prop-types';
+
+import { withStyles } from '@material-ui/core/styles';
+
+import LinkNoStyle from '../LinkNoStyle/LinkNoStyle';
+
+const styles = theme => ({
+    btn: {
+        fontFamily: theme.typography.fontFamily,
+        fontWeight: 500,
+    },
+});
+
+function ButtonBar(props) {
+    const { classes, isMobile } = props;
+
+    const PoppinLink = ({ to, ...props }) =>
+			<LinkNoStyle to={to}>
+				<Button fullWidth={isMobile} className={classes.btn} {...props} />
+            </LinkNoStyle>;
+    const BorderLink = ({ to, ...props }) =>
+        <LinkNoStyle>
+            <Button variant="outlined" fullWidth={isMobile} className={classes.btn} {...props} />
+        </LinkNoStyle>
+
+		const links = [
+			{
+				name: "home",
```

Comment:
These should be properly capitalized. Even though the CSS will make them appear all caps, we should keep the HTML semantically correct.

```suggestion
				name: "Home",
```